### PR TITLE
zyre 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by colstrom**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

Adds formula for current stable release of zyre, a library for
reliable group messaging over local area networks.

### CODE REVIEW

**Comment by dunn** [comment]
The test is failing:
```
==> zpinger
dyld: lazy symbol binding failed: Symbol not found: _zsys_ipv6
  Referenced from: /usr/local/Cellar/zyre/1.2.0/lib/libzyre.1.dylib
  Expected in: flat namespace

dyld: Symbol not found: _zsys_ipv6
  Referenced from: /usr/local/Cellar/zyre/1.2.0/lib/libzyre.1.dylib
  Expected in: flat namespace

**Comment by mrusme** [comment]
Ran `brew install "https://raw.githubusercontent.com/colstrom/homebrew-core/58f635f10041ae81f861604754be659d1f5faa86/Formula/zyre.rb"` locally, which succeeded. Thank you btw for this! 👍

File: Formula/zyre.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "zpinger"
```
Comment (bfontaine) [inline Formula/zyre.rb]: Please use the full path to the executable: `system bin/"zpinger"`.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by woodruffw** [comment]
Closing due to inactivity.

**Pull request closed without merging**